draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss

```diff
@@ -64,16 +71,20 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
   width: 100%;
   align-items: center;
   text-align: left;
-  background-color: white;
+  background-color: $color-white;
   border: none;
-  transition: background-color $ca-duration-rapid;
+  transition: background-color $animation-duration-rapid;
 
-  @include ca-padding(
-    $top: $ca-grid,
-    $end: $ca-grid,
-    $bottom: $ca-grid,
-    $start: $ca-grid
-  );
+  padding-top: $spacing-md;
+  padding-bottom: $spacing-md;
+  padding-right: $spacing-md;
+  padding-left: $spacing-md;
+
+  &[dir="rtl"],
+  [dir="rtl"] & {
+    padding-right: $spacing-md;
+    padding-left: $spacing-md;
+  }
```

Comment:
Possibly don't need this if the padding isn't changing in RTL layouts? assuming it doesn't get remove somehow 🤞

Reply:
Unless this is to ensure specificity 🤔